README.md

```diff
@@ -1,20 +1,157 @@
 
-# Fairness in Machine Learning - Mitigation algorithms
 
-A Python package that implements a variety of fairness-related algorithms to mitigate bias including:
+
+Fairlearn
+=============================================================
+Fairlearn is a machine learning fairness toolkit that makes it easier for anyone involved in the development of Artificial Intelligence (AI) systems to monitor and understand the impact of unfairness in their machine learning lifecycle, and use state-of-the-art techniques to mitigate the observed unfairness. Fairlearn repository contains an SDK and Jupyter notebooks with examples to showcase its use.
+
+- [Overview of Fairlearn](#intro)
+- [Target Audience](#target)
+- [Fairness Assessment Dashboard](#getting-started)
+- [Unfairness Mitigation Techniques](#models)
+- [Install Fairlearn](#install)
+- [Contributing](#contributing)
+- [Code of Conduct](#code)
+- [Build Status](#build-status)
+- [Additional References](#refs)
+
+
+
+ 
+# <a name="intro"></a>
+# Overview of Fairlearn
+
+We are at an age where many processes and applications have become or are becoming automated by AI systems. AI is the product of human processes and decisions used to create it, the data used to train it, and the environment used to test it.  The AI system can exhibit different, sometimes harmful, behaviors as a result of this process and systematically discriminate against people based on certain attributes protected by law (e.g., gender, ethnicity). Therefore, regulated industries, or any other industry with legal, moral and ethical responsibilities, need tools to ensure the fairness of models and the predictions they produce. 
+
+
+`Fairlearn` is an open source unfairness assessment and mitigation toolkit that provides business stakeholders, executives, developers, and data scientists with insights into the unfairness of their model predictions, and techniques to mitigate that. In particular, Fairlearn assists in 
+
+1) Raising awareness of inherent unfairness in model predictions among those involved in developing AI applications by demonstrating harms visited on vulnerable groups
+
+2) Providing state-of-the-art techniques to mitigate the observed unfairness
+
+
+At the assessment phase, Fairlearn provides a dashboard with a rich set of visualizations that uses a set of disparity and accuracy metrics and user-defined protected attributes (e.g., age, gender, ethnicity) to monitor the accuracy and fairness of a model’s predictions.
+
+
+At the mitigation phase, it incorporates state-of-the-art unfairness mitigation techniques to mitigate the unfairness observed at the previous assessment phase. Fairlearn’s mitigation stage supports regression and binary classification models trained on tabular data.
```

Comment:
I'm seeing "state-of-the-art unfairness mitigation techniques" repeated a lot.

And we might want "_currently_ supports" :-)